packages/mobile/src/analytics/ValoraAnalytics.ts

```diff
@@ -88,14 +90,28 @@ class ValoraAnalytics {
     return !__DEV__ && store.getState().app.analyticsEnabled
   }
 
-  startSession(
+  async startSession(
     eventName: typeof AppEvents.app_launched,
     eventProperties: AnalyticsPropertiesList[AppEvents.app_launched]
   ) {
     this.track(eventName, {
       deviceInfo: this.deviceInfo,
       ...eventProperties,
     })
+
+    if (Platform.OS === 'ios') {
```

Comment:
Should this happen before the `this.track` above?

Reply:
I wondered that too but thought it made more sense that this happened after the `app_launch` event.
Since the tracking permission request is tracked too now.